site/en/docs/webstore/manage/index.md

```diff
@@ -103,6 +103,52 @@ store item by adding `/reviews` at the end of your item’s URL:
 
 {% endAside %}
 
+## Provide user support
+
+To ensure the best user experience and build a great extension it's important to
+collect, evaluate, and follow-up on user feedback. You can do so by enabling user feedback in the
+CWS dashboard Account setting page.
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/ArMfJztL1OlP6UektUwb.png", alt="Enable User Feedback Support tab", width="800", height="185" %}
+
+Users will be able to ask questions, request features and report bugs in the **Support Tab** of your store item.
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/OBDYk7n5dBcaqs3z1HtL.png", alt="Store item Support tab", width="800", height="121" %}
+
+There are two ways in which you can manage your extension's user feedback.
+
+### Using the User Support tab
+
+In the **User Support Tab** of your item you can view, respond and manage user feedback. Use the *Type*
+dropdown to filter user input by
+feature request, bug report or question. You can assign a status to each request and respond to each inquiry. Each ticket includes the extension
+version, browser type and operating system to help you reproduce bugs
+more efficiently.
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/VvJFAHdrcMnUivTLm0kK.png", alt="Dashboard User
+Support Tab", width="800", height="269" %}
+
+{% Aside %}
+Currently, there are no notifications for this feature.
+
+- You will not be notified when a user posts a new request.
+- The user will not be notified when you post a response.
+
+{% endAside %}
+
+### Using a dedicated support site
+
+You can set up a dedicated support site for your users, so that the support link goes there instead of the default forum experience. This site can be anything you like, such as:
+
+* A discussion group using Google Groups or some similar service
```

Comment:
End each of these bulleted phrases with a period, as shown in https://developers.google.com/style/lists